chapter_introduction/index_vn.md

```diff
@@ -910,7 +916,12 @@ we will abuse language just a bit cn continue to describe
 our outputs and targets as real-valued numbers.
 -->
 
-*dịch đoạn phía trên*
+Để xác định một bài toán là *hồi quy* hay không, ta dựa vào đầu ra của nó.
+Chẳng hạn bạn đang thị sát thị trường cho một căn nhà mới.
+Bạn có thể đánh giá giá trị thị trường của một căn nhà khi biết trước những đặc trưng phía trên. Giá trị mục tiêu, giá bán của căn nhà, là một *số thực*.
+Nếu bạn còn nhớ định nghĩa chính thống của số thực, bạn có thể băn khoăn.
```

Comment:
```suggestion
Nếu bạn còn nhớ định nghĩa chính thống của số thực, bạn có thể băn khoăn.
```
mình vẫn thấy dịch từ formal ra chính thống nghe nó hơi sao sao. Nếu là "chuẩn" thì sao nhỉ?

Reply:
Mình cũng không thích từ "chính thống", nhưng "chuẩn" nghe standard quá. "Chuẩn" có thể hiểu là chỉ một, trong khi "chính thống" thì có thể nhiều.

Reply:
How about "chính thức", "hình thức", 
![image](https://user-images.githubusercontent.com/6524339/71060806-d5e58580-21a9-11ea-9110-05ab39b16b3f.png)

Reply:
Mấy cái từ điển chuyên ngành tiếng Việt chỉ để tham khảo thôi, đừng có tin. Bọn này dịch có được review kỹ như mình đang làm đâu. Nếu có tài liệu nào sử dụng các cụm này rồi dẫn nguồn vào đây thì tốt hơn.

Từ "chính thức" ok, nhưng "hình thức" không hề rõ nghĩa.

Reply:
em thấy từ "formal" trong "formal proof/definition/etc." thường là nói tới định nghĩa toán học, không nhất thiết phải là về toán nhưng phải dùng kí hiệu toán học để mô tả cho thật chặt chẽ.
Mình có thể "đi tắt đón đầu" từ "formal" bằng "toán học" luôn được không?
```suggestion
Nếu bạn còn nhớ định nghĩa toán học của số thực, bạn có thể băn khoăn.
```

Reply:
em thấy từ "formal" trong "formal proof/definition/etc." thường là nói tới định nghĩa toán học, không nhất thiết phải là về toán nhưng phải dùng kí hiệu toán học để mô tả cho thật chặt chẽ.
Mình có thể "đi tắt đón đầu" từ "formal" bằng "toán học" luôn được không?
```suggestion
Nếu bạn còn nhớ định nghĩa toán học của số thực, bạn có thể băn khoăn.
```

Reply:
@lkhphuc ok, mình thấy từ này không quan trọng, dịch là "toán học" nghĩa rất sát trong trường hợp này rồi. @ngcthuong thấy sao?

Reply:
Mình thấy ok nhưng chỉ trong trường hợp này thôi, vì ở đây là số thực. 

Giả dụ trường hợp khác ghi định nghĩa "formal" của data science chẳng hạn, thì nó không có nghĩa là "toán học". Nên dịch là "toán học" cần cân nhắc trường hợp cụ thể thôi.

Reply:
Uh, trong trường hợp này thôi, Ở trường hợp khác lại phải tìm cách dịch khác. Nhưng theo mình đây không phải từ khoá nên không quan trọng.